deep-learning/_papers/2017-02-28-you-only-look-once.md

```diff
@@ -0,0 +1,52 @@
+
+
+```
+---
+layout: review
+title:  You Only Look Once: Unified, Real-Time Object Detection (YOLO)
+author: Frédéric Branchaud-Charron
+link:   https://arxiv.org/abs/1506.02640
+tags:   convolutional-networks deep-learning object-detection bounding-boxes
+---
+```
+
+YOLO v1
```

Comment:
Same as SSD:

* Remove both \`\`\`
* Same thing as above for quotes around title and name
* Remove "YOLO v1"